docs/contract-test-collection.md

```diff
@@ -1493,6 +1493,43 @@ storage (pair (pair (bool %areyouthere) (int %integer))
 </TabItem>
 </Tabs>
 
+## SaplingContract
+
+[KT1CDenBWcgWjNZULc9GbJRTnQZQXYWrVT7k](https://better-call.dev/ithacanet/KT1CDenBWcgWjNZULc9GbJRTnQZQXYWrVT7k/code)
+
+Sapling is a protocol enabling privacy-preserving transactions of fungible tokens in a decentralised environment. The example contract used
+in Taquito Integration Tests is a single-state sapling contract. It features the Michelson instruction "SAPLING_VERIFY_UPDATE".
+A sapling_state is represented by an integer value in the contract storage. 
+
+#### Entrypoints:
+
+- default
+
+<Tabs
+defaultValue="michelson"
+values={[
+{label: 'Michelson Storage', value: 'michelson'},
+{label: 'Taquito Storage', value: 'taquito'},
+]}>
+<TabItem value="michelson">
+
+```js
+storage (sapling_state 8);
+```
+
+</TabItem>
+  <TabItem value="taquito">
+
+```js
```

Comment:
the storage doesn't seem right.

It can be:
```
import { SaplingStateValue } from @taquito/michelson-encoder
storage: SaplingStateValue
```

Reply:
fixed

Reply:
My comment was for the taquito tab, not michelson one.

The following storage does not match :
```
`(Pair (Pair True 0)
      (Pair ""
            { Elt 0x00006b82198cb179e8306c1bedd08f12dc863f328886 "Alice" ;
              Elt 0x0000b2e19a9e74440d86c59f13dab8a18ff873e889ea "HEllo!" }))`,
```